doc/source/conf.py

```diff
@@ -183,7 +183,7 @@
 scv_banner_greatest_tag = True
```

Comment:
So we need:

-scv_banner_greatest_tag = True
+scv_banner_main_ref = 'stable/4.0'